16.md

```diff
@@ -10,7 +10,7 @@ Relays may decide to allow replaceable and/or ephemeral events.
 
 Regular Events
 ------------------
-A *regular event* is defined as an event with a kind `1000 <= n < 10000`.
+A *regular event* is defined as an event with a kind `0 <= n < 10000`.
```

Comment:
This should mention that 0, 3, and 41 are replaceable events.

Reply:
I added mention of those events.